exo-go/src/service_config_helpers/index.go

```diff
@@ -0,0 +1,26 @@
+package serviceConfigHelpers
+
+import (
+	"io/ioutil"
+	"log"
+	"path"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+)
+
+// GetServiceConfig reads service.yml of the role service and returns
+// the serviceConfig object
+func GetServiceConfig(location string) types.ServiceConfig {
```

Comment:
Maybe rename `location` -> `serviceFilePath`? Based on the code I believe the location is the root directory of the service and ideally the name is able to reflect that